test/libsolidity/syntaxTests/natspec/doctring_parameter_colon.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    /**
+     * @dev    : token transfer
+     * @param _from : from address
+     * @param _to   : to address
+     * @param _value: token transfer amount
```

Comment:
Won't this line result in a buggy parameter name? Because there is no space followed by a tab character followed by a colon?

Reply:
I think the parameter name in this case is `_value:`.

Reply:
The parameter name in this case is `_value` and the description is `: token transfer amount`. After looking at the NatSpec format again, I think that my change is only partially correct. We could allow the colon (as well as other special characters / usual delimiters, such as `=` etc.) and make it part of the description? Like that, it should be obvious to developers that those delimiters are not valid.

Reply:
I think I misunderstood the original problem setting. The thing is that `:` has no special meaning in natspec. We could search for the first non-identifier character instead of the first space or tab, though.

Reply:
Yes, exactly. I'll do that.